fix(doc): update text around creating/setup of DB (#322)
temporalio/helm-charts: README.md

In this case, create and configure temporal databases on your PostgreSQL host with `temporal-sql-tool`. The tool is part of [temporal repo](https://github.com/temporalio/temporal), and it relies on the schema definition, in the same repo.

> If the user temporal will connect with does not have permissions to create the databases, create them and skip the
> `create-database` commands.  For example, with PostgreSQL, one could:

Reviewer: ```suggestion
If a user does not have permission to create the databases, create them and skip the `create-database` commands. For example, with PostgreSQL, one could:
```
